locales/en.js

```diff
@@ -352,8 +351,10 @@ export default {
   commonActions: 'Common actions',
 
   //benefit task headings
-  commonActions: 'Explore common actions',
-  paymentsTaxesAccount: 'Payments, taxes and account information',
+  viewMore: 'Explore common actions',
```

Comment:
I think this could use a better prop name.

Reply:
changed